pkg/cmd/component/bootstrap_new.go

```diff
@@ -134,9 +138,13 @@ func nonInteractiveModeBootstrapNew(io *BootstrapNewParameters) error {
 }
 
 // Checking the mandatory flags & reusing missingFlagErr in Bootstrap.go
-func checkMandatoryFlags(flags map[string]string) error {
+func CheckMandatoryFlags(flags map[string]string) error {
```

Comment:
Could this function be moved to some other common file?  The other commands call this so it seems odd to have to reference the 'bootstrap-new' package.

Reply:
Yeah we can shift this to validate.go